app/views/index_of_poets/items/show.html.erb

```diff
@@ -1,11 +1,16 @@
 <% if @res %>
-  <h1 class="item_title"><%= @res["alternative"] %></h1>
+  <h1 class="item_title"><%= @res["person_alternate_name_k"] %></h1>
   <h2 class="item_subtitle">
     <% if @res["people"].present? %>
       Alternate names: <%= @res["people"] %>
     <% end %>
   </h2>
 
+  <% if @res["subcategory"].include?("In the News") %>
+    <h3>Poet in the news</h3>
```

Comment:
Simple change here - "Poet in the News" since our other labels (Selected Works, etc.) are title case

Reply:
Fixed